source/slang/slang-ir-autodiff-transcriber-base.cpp

```diff
@@ -486,15 +316,20 @@ IRType* AutoDiffTranscriberBase::_differentiateTypeImpl(IRBuilder* builder, IRTy
     }
 
     case kIROp_TupleType:
+    case kIROp_TypePack:
```

Comment:
Yong's note: here we implement the IR-level differenetiation rules for tuple types and type packs.